yantra/JointValues[] Q
float64[] T
---
yantra/TrajCoefficient[] coeff
